operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -16,7 +16,7 @@ This document describes the configuration of a Kubernetes NCN image. The same st
     ```bash
     ARTIFACT_VERSION=<artifact-version>
 
-    cray artifacts get boot-images "k8s/${ARTIFACT_VERSION}/rootfs" "./${ARTIFACT_VERSION}-rootfs"
+    cray artifacts get boot-images "k8s/${ARTIFACT_VERSION}/filesystem.squashfs" "./${ARTIFACT_VERSION}-filesystem.squashfs"
```

Comment:
This needs to be `rootfs` in S3.

Reply:
After the pre-signed URL bucket work went in this changed to `rootfs`, that went into 1.3.0-beta.47 and newer (including 1.3.0-rc.2).

1.3.0-beta.46 and older (including 1.3.0-rc.1) used `filesystem.squashfs`.

Reply:
```suggestion
    cray artifacts get boot-images "k8s/${ARTIFACT_VERSION}/rootfs" "./${ARTIFACT_VERSION}-filesystem.squashfs"
```